<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LostArkNote</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QScrollArea{
	border:1px solid #000000;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="styleSheet">
       <string notr="true">Character:hover#Character,
ToDoSingleWidget:hover#ToDoSingleWidget
{
        background-color: rgb(215, 215, 215);
        border-radius:10px;
}
</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="m_widget_CharacterContainer">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>302</width>
          <height>382</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="m_verticalLayout_character">
           <item>
            <spacer name="m_verticalSpacer_character">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>68</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QScrollArea" name="scrollArea_2">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="m_widget_toDoContainer">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>302</width>
          <height>382</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="m_verticalLayout_toDo">
           <item>
            <spacer name="m_verticalSpacer_toDo">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>108</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>636</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>开始</string>
    </property>
    <addaction name="add_character"/>
    <addaction name="separator"/>
    <addaction name="clean_finished_status"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="open_config_dir"/>
    <addaction name="open_app_config_dir"/>
    <addaction name="separator"/>
    <addaction name="update_config"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="add_character">
   <property name="text">
    <string>添加角色</string>
   </property>
  </action>
  <action name="clean_finished_status">
   <property name="text">
    <string>周三啦</string>
   </property>
  </action>
  <action name="open_config_dir">
   <property name="text">
    <string>打开角色配置文件夹</string>
   </property>
  </action>
  <action name="update_config">
   <property name="text">
    <string>更新配置文件</string>
   </property>
  </action>
  <action name="open_app_config_dir">
   <property name="text">
    <string>打开程序配置文件夹</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
